givenergy_modbus/pdu/transparent.py

```diff
@@ -24,7 +24,7 @@ class TransparentMessage(BasePDU, ABC):
 
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
-        self.slave_address = kwargs.get("slave_address", 0x32)
```

Comment:
I assume this is a deliberate move?  I've never been quite sure which addresses are best.  I'm actually wondering whether there's a case for using 0x31 rather than 0x32 for the plant's cache of inverter registers. That keeps them in a separate dictionary from battery #0 which seems like a good thing.

Reply:
Yes, and 0x32 will get sent to GE cloud, so using 0x31 stops unnecessary data sent to GE servers.